application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarkPaginatorInteractor.java

```diff
@@ -0,0 +1,480 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
+import net.dv8tion.jda.api.entities.emoji.Emoji;
+import net.dv8tion.jda.api.events.interaction.ModalInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.command.GenericCommandInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.component.ButtonInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.component.SelectMenuInteractionEvent;
+import net.dv8tion.jda.api.interactions.components.ActionRow;
+import net.dv8tion.jda.api.interactions.components.ComponentInteraction;
+import net.dv8tion.jda.api.interactions.components.LayoutComponent;
+import net.dv8tion.jda.api.interactions.components.buttons.Button;
+import net.dv8tion.jda.api.interactions.components.selections.SelectMenu;
+import net.dv8tion.jda.api.interactions.components.selections.SelectOption;
+
+import org.togetherjava.tjbot.commands.UserInteractionType;
+import org.togetherjava.tjbot.commands.UserInteractor;
+import org.togetherjava.tjbot.commands.componentids.ComponentIdGenerator;
+import org.togetherjava.tjbot.commands.componentids.ComponentIdInteractor;
+import org.togetherjava.tjbot.commands.utils.Arraylizable;
+import org.togetherjava.tjbot.db.generated.tables.records.BookmarksRecord;
+
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.StringJoiner;
+import java.util.function.Predicate;
+import java.util.function.UnaryOperator;
+
+public final class BookmarkPaginatorInteractor implements UserInteractor {
+
+    private static final int ENTRIES_PER_PAGE = 10;
+    private static final Emoji BUTTON_PREV_EMOJI = Emoji.fromUnicode("⬅");
+    private static final String BUTTON_PREV_NAME = "button-prevoius";
+    private static final Emoji BUTTON_NEXT_EMOJI = Emoji.fromUnicode("➡");
```

Comment:
can we just use the unicode character?? not necessary but i just like that a lil bit more

feel free to just resolve this if u don't want to change

Reply:
Let me check out your unicode buttons

Reply:
![image](https://user-images.githubusercontent.com/54677650/198837420-b2b436d2-9555-4fb0-9b14-aa12feb36156.png)

That looks horrible imo

Reply:
We can allso change the button color to green if that is what you want

Reply:
for me both are the same size maybe some issue on ur side, see the images i put on my pr

Reply:
Huh weird. My android phone shows them as other emojis. Im gonna spin up a windows VM to test if its operating system dependant

Reply:
![image](https://user-images.githubusercontent.com/54677650/198838816-f0dcb42c-7cd4-4244-9c74-e834af56517f.png)
 On windows they are pretty much just a line... We should also go with emojis for your navigation buttons.

Reply:
![image](https://user-images.githubusercontent.com/54677650/198890571-c9497ad3-9038-4d20-9c35-272adf295eed.png)

@Taz03 how do you feel about changing the button color to green?

Reply:
looks a bit unusual to me

Reply:
So back to blue?